extensions/wikia/Email/EmailMobileBadges.class.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class EmailMobileBadges
+ *
+ * @package Email
+ */
+class EmailMobileBadges {
+
+	static private $badges = [
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$badges" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1068)